### ISSUE TITLE
Frontend - Move About components and Add border to detail box in About section 

### ISSUE THREAD

**Issue by minhngo3818** [Computer-Science-Club-OCC/Computer-Science-Club-OCC.github.io#129]
In this issue, there are two tasks that need to be fulfilled

Task 1: Move ```about``` and ```contact-us``` folders to the directory ```~/client/src/components/home.```

Task 2: Add a border to the detail box in the About section. This task requires adding style to the CSS class ```about-text```. The style is under the file ```~/about/about.css```
Use only CSS for simplicity. (A follow-up issue will be posted once this issue is merged into the main branch )

Styling detail: for ```about-text```
- add border: 2px
- padding all: 4px
- border color: #ff9617
- add to group hover effect: when the user hovers to the area of ```<div class="about-grid">``` image moves diagonally and a line under ```<h1>``` expand ("Who we are"). the border must appear in the same way when hover to that area.

Demo
![Detail (1)](https://user-images.githubusercontent.com/78233689/229657542-cb430f24-50e2-4364-b6d6-b0ca012200a3.png)

White frame:
<img width="825" alt="Untitled" src="https://user-images.githubusercontent.com/78233689/229657276-01b99719-f85e-415a-a29d-950fca5cf8a3.png">
